proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -865,6 +899,18 @@ This procedure takes a channel value `value`, and returns the special value
 * If `value` is the Javascript value `null`, return the unquoted Sass string
   `none`.
 
+### Parsing a Clamped Channel Value
+
+This procedure takes a channel value `value` and an inclusive range of `minimum`
+and `maximum`. It asserts the value is in the range, and returns the special
+value `none` if the value is `null`.
+
+* If `value` is less than `minimum`, throw an error.
```

Comment:
These should be fuzzy comparisons. If the user passes `lightness: 100.000000000001`, that should be treated as `lightness: 100`.

Reply:
Don't treat the Dart Sass implementation as authoritative—it's lagging behind the spec in a few crucial ways. Currently, the _only_ clamped channels are `lightness` and `alpha`, and so those should be the only ones that throw out-of-range errors (although that may change as the result of https://github.com/w3c/csswg-drafts/issues/9484).